jython/DCC-EX/SendDCCppMessages.py

```diff
@@ -1,4 +1,4 @@
-# Example of sending various messages to a DCCpp connection.
+# Example of sending various messages to a DCCpp/DCC-EX connection.
```

Comment:
Just checking. Could any user use this file without copying it to their preferences folder? If that's the case, moving the file could break things for the users.

Reply:
> Just checking. Could any user use this file without copying it to their preferences folder? If that's the case, moving the file could break things for the users.

They could, but the commands are just examples, not something that would make sense in a live environment. So the user will need to modify it to use it.